kyber/objects/secret.py

```diff
@@ -41,10 +41,7 @@ def __unicode__(self):
         return unicode(repr(self))
 
     def has_key(self, k):
-        return self.obj['data'].has_key(k)
-
-    def pop(self, k, d=None):
-        return self.obj['data'].pop(k, d)
+        return k in self.obj['data']
```

Comment:
Why remove `pop`, and why not pass through the `has_key` method?  The intent of these methods is to mirror the standard interface of a `dict` (and `self.obj['data']` is a dict).